src/components/Synthetics/GmSwap/GmSwapBox/GmDepositWithdrawalBox/GmDepositWithdrawalBox.tsx

```diff
@@ -273,37 +289,30 @@ export function GmSwapBoxDepositWithdrawal(p: GmSwapBoxProps) {
     [marketInfo, tokensData, isGlvMarket, marketTokensData, marketsInfoData, isPair]
   );
 
-  const bestGmPoolAddress = useBestGmPoolAddressForGlv(isDeposit, marketInfo, selectedGlvGmMarket);
-
-  useEffect(() => {
-    if (!selectedGlvGmMarket && bestGmPoolAddress) {
-      onSelectGlvGmMarket?.(bestGmPoolAddress);
-    }
-  }, [bestGmPoolAddress, onSelectGlvGmMarket, selectedGlvGmMarket]);
+  const vaultInfo = isGlvMarket ? marketInfo : undefined;
 
   /**
-   * Here and following we use `underlyingGmMarket` to get the correct market info for GLV markets
+   * Here and following we use `targetGmMarket` to get the correct market info for GLV markets
    * because we need to use actual GM market data to calculate fees, errors and so on,
    * but here marketInfo may be a Vault market
    */
-  const { underlyingGmMarket, underlyingGmMarketToken, marketTokenAmount, marketToken } = useMemo(() => {
-    const realGmMarket = isGlvMarket
+  const { targetGmMarket, targetGmMarketToken, marketTokenAmount, marketToken } = useMemo(() => {
```

Comment:
we now have gmToken, marketToken and gmMarketToken which reference same thing.

let's agree to stick to just one naming?
let's rename all to marketToken, as this is dominant usage

I won't highlight every occasion of gm/gmMarket as it is pretty obvious to find but I expect to have it fixed

Reply:
remove all these targetGmMarket, gmAddress etc, now there's only marketToken, marketTokenAmount (which is GM) and glvToken and glvTokenAmount (which is GLV)